src/HTML5toTouch.ts

```diff
@@ -1,25 +1,31 @@
+import type { MultiBackendOptions } from "dnd-multi-backend";
+import { PointerTransition, TouchTransition } from "dnd-multi-backend";
+import type { HTML5BackendOptions } from "react-dnd-html5-backend";
 import { HTML5Backend } from "react-dnd-html5-backend";
+import type { TouchBackendOptions } from "react-dnd-touch-backend";
 import { TouchBackend } from "react-dnd-touch-backend";
 
-import {
-  TouchTransition,
-  PointerTransition,
-  MultiBackendOptions,
-} from "dnd-multi-backend";
-
-export const HTML5toTouch: MultiBackendOptions = {
-  backends: [
-    {
-      id: "html5",
-      backend: HTML5Backend,
-      transition: PointerTransition,
-    },
-    {
-      id: "touch",
-      backend: TouchBackend,
-      options: { enableMouseEvents: true },
-      preview: true,
-      transition: TouchTransition,
-    },
-  ],
+export const getBackendOptions = (
```

Comment:
Rename the file `getBackendOptions.ts`.

Reply:
Do you mind if I move the file into utils folder after renaming?

Reply:
Yes, it is! I think it's better that way!